### DISCUSSION TITLE
Idea for radio immersion

### DISCUSSION THREAD

**Discussion by Tyburnbomb** [IDI-Systems/acre2#1185] [Ideas]
Would there be any possible way to include some surrounding in game sounds through the radio, for example gun shots, explosions, possibly slight vehicle noise, etc. Got this idea from a guys mic picking up his gunshots and in game noise through his headset while he was speaking on radio.

**Comment by jonpas** [comment]
We want it, but it's not easily possible in the current system. Maybe next iteration of ACRE. :)
